shared/src/business/entities/EntityConstants.ts

```diff
@@ -218,6 +218,13 @@ export const DOCUMENT_SERVED_MESSAGES = {
   SELECTED_CASES: 'Document served to selected cases in group.',
 };
 
+export const VALIDATION_ERROR_MESSAGES = {
+  END_DATE_IN_THE_FUTURE_ERROR_MESSAGE:
+    'End date cannot be in the future. Enter a valid date.',
+  END_DATE_PRIOR_TO_START_DATE_ERROR_MESSAGE:
+    'End date cannot be prior to Start Date. Enter a valid end date.',
```

Comment:
want to make sure with UX that "Start Date" should be capitalized!

Reply:
Checked in with UX. Corrected messaging